frontend/packages/db-structure/src/parser/prisma/index.test.ts

```diff
@@ -560,6 +560,140 @@ describe(_processor, () => {
       expect(value).toEqual(expectedTables)
     })
 
+    it('@@map', async () => {
+      const { value } = await processor(`
+        model User {
+          id    Int     @id @default(autoincrement()) @map("_id")
+          posts Post[]
+          email String  @unique @map("raw_email_address")
+          role  Role    @default(USER)
+    
+          @@map("users")
+        }
+    
+        model Post {
+          id     Int   @id @default(autoincrement())
+          user   User  @relation(fields: [userId], references: [id])
+          userId Int   @map("raw_user_id")
+    
+          @@map("posts")
+        }
+
+        enum Role {
+          USER
+          ADMIN
+        }
+      `)
+
+      const expectedTables = aSchema({
+        tables: {
+          User: aTable({
+            name: 'User',
```

Comment:
The `@@map` specifies the actual database table name, which we want changed.

```suggestion
          users: aTable({
            name: 'users',
```

`@map` is supported because it is a column name mapping, but I thought this one was not.

Reply:
I see. let me update the table name.